.github/workflows/build.yml

```diff
@@ -93,15 +93,27 @@ jobs:
 
   tests:
     needs: build
-    name: ${{ matrix.grunt-cmd }}
+    name: ${{ matrix.grunt-cmd }}_${{ matrix.suite ||'' }}
     runs-on: ubuntu-22.04
     env:
       NODE: 16.x
 
     strategy:
       fail-fast: false
       matrix:
-        grunt-cmd: ['ci-webdriver-default', 'ci-webdriver-standard', 'ci-e2e-integration', 'ci-webdriver-default-mobile']
+        grunt-cmd: ['ci-webdriver-standard', 'ci-e2e-integration', 'ci-webdriver-default-mobile']
+        suite: [all]
```

Comment:
Correct me if I am wrong, but this means that it will run all the e2e tests everything time that we create a build, and I think that the main idea of this change was to define a set of e2e tests that are "mandatory" or "critical" for each build to be created.

I think that we need to create a suite with a combination of tests that we want to be run every single time that a new build is created, for example:

- new-lineage-structure.wdio-spec
- search-contacts.wdio-spec
- pregnancy.wdio-spec
- login-logout.wdio-spec
- navigation.wdio-spec
- enabled-languages.wdio-spec
- add-user.wdio-spec
- ... any other test that the team thinks is valuable for each time that we create a build

Reply:
I think we want to run all tests for every build.

Reply:
Perfect, thank you!